examples/abitest/tests/src/tests.rs

```diff
@@ -32,8 +33,8 @@ const BACKEND_MANIFEST: &str = "../module_1/rust/Cargo.toml";
 const FRONTEND_WASM_NAME: &str = "abitest_0_frontend.wasm";
 const BACKEND_WASM_NAME: &str = "abitest_1_backend.wasm";
 
-fn build_wasm() -> std::io::Result<Vec<(String, Vec<u8>)>> {
-    Ok(vec![
+fn build_wasm() -> std::io::Result<HashMap<String, Vec<u8>>> {
+    Ok([
```

Comment:
I think this would read more nicely with `maplit`, and I don't mind having it as a dependency (it is very lightweight, and very widespread already). Your choice anyways.

Reply:
Done.